packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -92,4 +134,9 @@ export interface TerminalWidgetOptions {
      * Terminal id. Should be unique for all DOM.
      */
     readonly id?: string;
+
+    /**
+     * Terminal actions. Represents buttons, which locate on terminal toolbar.
+     */
+    readonly terminalActionItems?: TerminalActionItem[];
```

Comment:
(nit) How about `actionItems`? it will be less verbose and it is already clear that they are for terminals from `TerminalWidgetOptions`